src/load/azext_load/data_plane/utils/utils.py

```diff
@@ -473,6 +503,27 @@ def create_or_update_test_with_config(
         new_body["loadTestConfiguration"]["splitAllCSVs"] = yaml_test_body[
             "loadTestConfiguration"
         ]["splitAllCSVs"]
+
+    logger.debug("autostop_criteria: %s", autostop_criteria)
+    logger.debug("yaml_test_body: %s", yaml_test_body["autoStopCriteria"])
```

Comment:
itentional to keep these values or plannning to remove before merging the PR

Reply:
we do keep some debug logs generally, but these ones can be removed so will do